libraries/stdlib/common/src/generated/_Arrays.kt

```diff
@@ -12710,6 +12865,8 @@ public inline fun <S, T : S> Array<out T>.reduceRight(operation: (T, acc: S) ->
 
 /**
  * Accumulates value starting with last element and applying [operation] from right to left to each element and current accumulator value.
+ * 
+ * @sample samples.collections.Collections.Aggregates.reduceRightOrNull
```

Comment:
This sample is for `reduceRightOrNull` and doesn't relate to `reduceRight`. Note that the samples for `reduceRight` have been already submitted in the PR #2867.

Reply:
Oops, yes, you are right, misplaced! Updated the PR :)